tests/components/fritz/test_button.py

```diff
@@ -42,6 +42,9 @@ async def test_button_setup(
         assert await hass.config_entries.async_setup(entry.entry_id)
         await hass.async_block_till_done()
 
+    states = hass.states.async_all()
+    assert len(states) == 5
+
```

Comment:
Could/Should this not be included as part of "snapshot_platform" directly?

Reply:
the idea was to add this in a later PR, since it would mean to update all `*.ambr` (_snapshot_) files, which are created by the snapshot_platform helper and these are a lot. But yeah ... after some thinking about, it would make sense to do it right now and those skip the current PR 🤔

Reply:
I may have a solution to our woes...
See latest on #128162